### PR TITLE
Split dependabot into two groups

### PR DESCRIPTION
**Pull request by dmlloyd**
This should fix the timeout issues. The two groups are equally-sized and just divided in the middle. The time is offset by one hour, since the timeout is less than that.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -96,6 +96,22 @@ updates:
       - dependency-name: org.wiremock:wiremock
       - dependency-name: org.wiremock:wiremock-standalone
       - dependency-name: uk.co.automatictester:wiremock-maven-plugin
+    ignore:
+      # Major/minor upgrades require more work and synchronization, we do them manually.
+      # Only use dependabot for micros (patch versions).
+      - dependency-name: org.hibernate.*:*
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
```
Comment (gastaldi) [inline .github/dependabot.yml:103]: I think this line needs to be removed?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -96,6 +96,22 @@ updates:
       - dependency-name: org.wiremock:wiremock
       - dependency-name: org.wiremock:wiremock-standalone
       - dependency-name: uk.co.automatictester:wiremock-maven-plugin
+    ignore:
+      # Major/minor upgrades require more work and synchronization, we do them manually.
+      # Only use dependabot for micros (patch versions).
```
Comment (gastaldi) [inline .github/dependabot.yml:101]: This too

**Comment by dmlloyd** [comment]
Note: I'm not certain whether multiple of the same `package-ecosystem` is supported or not. I figure, there's at least one way to find out...

File: .github/dependabot.yml (hunk 1 above)
Comment (dmlloyd) [reply .github/dependabot.yml:103]: Why? Doesn't it apply to [lines 66-70](https://github.com/quarkusio/quarkus/pull/40276/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R66-R70) above?

Comment (gastaldi) [reply .github/dependabot.yml:103]: Ah my bad, didn't know you could do that in the `ignore`: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore

**Review by gsmet — APPROVED** [review]
LGTM!

I have a question though: why didn't I think about doing that months ago...

Let's how it goes in the next few days.

**Comment by dmlloyd** [comment]
Uh oh!

> Dependabot encountered the following error when parsing your .github/dependabot.yml:
> 
> Update configs must have a unique combination of 'package-ecosystem', 'directory', and 'target-branch'
> Please update the config file to conform with Dependabot's specification.

**Comment by dmlloyd** [comment]
Maybe one half could have `target-branch: main` and the other not given?

**Comment by gastaldi** [comment]
That's weird, you have tested in your branch before submitting, right? 😅

**Comment by dmlloyd** [comment]
> That's weird, you have tested in your fork before submitting, right? 😅

I guess I didn't realize that was possible!

**Comment by dmlloyd** [comment]
> Maybe one half could have `target-branch: main` and the other not given?

I guess that won't work :( Oh well, revert time.

**Pull request merged**